### ISSUE TITLE
Feature request : multiple repository

### ISSUE THREAD

**Issue by Anthares** [pgbackrest/pgbackrest#841]
Dear pgbackrest team,

First of all, thanks for this neat piece of software :)

I'm filling this issue because I'm currently improving our DR plan.
Part of it implies copy of backup to external site.

Since pgbackrest documentation suggest multiple repository support in future release, I tought it might be good to bring our use case to your attention.

Our plan is currently to rsync content of local repo to external site, but I find the idea somehow flawed since it imply running rsync periodically on archive which could result in partial/corrupted copy at DR site.
It would be nice to be able to have pgbackrest asynchronously duplicating writes to another local filesystem (mounted from nfs, glusterfs, ...) or even another pgbackrest repository.

Does it seems something doable ?
I would be surprised beeing the only one with this use case.

Thanks for you time,
Regards,

Nicolas

**Comment by dwsteele** [comment]
It is certainly the plan to support multiple repositories, but we won't be able to work on that until the C migration is complete.  Even then we don't have a set schedule.

Rsync'ing the repo is a perfectly safe thing to do, though.  It was designed with the intention of being used with rsync and enterprise backup tools.  You may be missing later data based on your sync interval, but you should not see corruption.

**Comment by Anthares** [comment]
Hello David,

Thanks for your answer :)

I understand you cannot change all your plans for a single request. We will go with a workaround for now and wait for the multiple repositories to be available.

I'm pleased to hear that rsync is safe, but we realised Rsync was a bit to slow to cope with archives throughput (one of our database is producing roughly 100GB a day).
Most of time was lost by rsync comparing the state of the second repo to know what should be done.
So we are trying another solution based on lsyncd (which uses inotify in background) to sync only the files that have been changed/add.

Regards,

Nicolas
